### PR TITLE
COG-747 getUser Service Response Contract Change

### PR DESCRIPTION
**Pull request by sorych**
### JIRA Issue Link
- [COG-747 getUser Service Response Contract Change](https://osi-cwds.atlassian.net/browse/COG-747 )

### Tests
- [ ] I have included unit tests 
- [x] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

### Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: src/main/java/gov/ca/cwds/idm/dto/UserByIdResponse.java [hunk 1]
```diff
@@ -4,65 +4,26 @@
 import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
 import java.io.Serializable;
-import java.util.List;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 public class UserByIdResponse implements Serializable {
 
   private static final long serialVersionUID = 2528332054070849802L;
```
Comment (dmytro-palczewski-osi) [inline src/main/java/gov/ca/cwds/idm/dto/UserByIdResponse.java]: serialVersionUID should probably be changed

Comment (sorych) [reply src/main/java/gov/ca/cwds/idm/dto/UserByIdResponse.java]: good catch, done!

Comment (sorych) [reply src/main/java/gov/ca/cwds/idm/dto/UserByIdResponse.java]: @dmytro-palczewski-osi PR updated, please check

**Comment by sorych** [comment]
please do not merge this until UI is ready for the contract change

**Comment by sorych** [comment]
WIP

**Pull request merged**